BasicSample/Assets/GrabAndThrow/Scripts/GrabAndThrowManager.cs

```diff
@@ -53,12 +53,33 @@ void Update()
 			UpdateForHand(XRNode.RightHand, m_rightHandData);
 		}
 
+		private bool? IsTapping(InputDevice device)
```

Comment:
super nit: We could rename this to `TryGetIsTapping` so it's a little more clear it can fail, plus it'll match all the other TryGetFeatureValue calls

Reply:
addressed in the last commit